index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css/program.css">
+    <title>Music Extravaganza</title>
+</head>
+<body> 
+      <header>
+        <nav class="nav1">
+          <a class="nav-link" href="#">Logout</a>
+          <a class="nav-link" href="#">My Page</a>
+          <a class="nav-link" href="#">English</a>
+          <a class="nav-link" href="#" aria-label="twitter" ><i id="twitter-icon" class="fa fa-twitter text-white"></i></a>
+          <a class="nav-link" href="#" aria-label="facebook"><i id="facebook-icon" class="fa fa-facebook text-white"></i></a>
+        </nav>
+        <nav class="nav2">
+          <div class="navlogo"><a href="#"><img src="images/Logo.png" alt="logo"></a></div>
+          <div class="navcontent">
+            <a class="nav-item" href="about.html">About</a>
+            <a class="nav-item" href="#mainprogram">Program</a>
+            <a class="nav-item" href="#">Join</a>
+            <a class="nav-item" href="#sponsor">Sponsor</a>
+            <a class="nav-item" href="#">News</a>
+            <a id="bordered-link" class="nav-item" href="#">CC Compaign</a>
+          </div>
+        </nav>
+      </header>
+      <section class="start">
+        <button class="mobile-menu" type="button"><img src="https://img.icons8.com/ios/50/000000/menu--v4.png" alt="hamburger-menu"/></button>
+        <div class="mobile-menu-list">
+          <a class="nav-item, nav-link" href="about.html">About</a>
+          <a class="nav-item, nav-link" href="#mainprogram">Program</a>
+          <a class="nav-item, nav-link" href="#">Join</a>
+          <a class="nav-item, nav-link" href="#sponsor">Sponsor</a>
+          <a class="nav-item, nav-link" href="#">News</a>
+          <a class="nav-item, nav-link" href="#">CC Compaign</a>
+        </div>
+        <h1 class="world">"Hello! Sharing world"</h1>
+        <h2 class="title">MUSIC FESTIVAL<br>EXTRAVAGANZA<br>2022</h2>
```

Comment:
- [ ] It seems the issue is still not resolved. as shown from this screen-shot:

![image](https://user-images.githubusercontent.com/51296741/153727585-fe7f9dfa-53d0-4b83-9df3-27daa967aeab.png)
It will be nice to fix it.